### PR TITLE
Update module with the latest PW lib. Add delivery confirmation api

### PR DESCRIPTION
**Pull request by vukhacdiep**

### CODE REVIEW

File: src/modules/gateways/callback/paymentwall.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+$ipsWhitelist = array(
+    '174.36.92.186',
+    '174.36.96.66',
+    '174.36.92.187',
+    '174.36.92.192',
+    '174.37.14.28'
+);
+require_once("../../../includes/api/paymentwall_api/lib/paymentwall.php");
```
Comment (davex69) [inline src/modules/gateways/callback/paymentwall.php]: remove

File: src/modules/gateways/callback/paymentwall.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    die("Module Not Activated");
+}
+
+$ipsWhitelist = array(
+    '174.36.92.186',
+    '174.36.96.66',
+    '174.36.92.187',
+    '174.36.92.192',
+    '174.37.14.28'
+);
```
Comment (sevastyanovio) [inline src/modules/gateways/callback/paymentwall.php]: remove

File: src/modules/gateways/callback/paymentwall.php [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    'public_key' => $GATEWAY['appKey'], // available in your Paymentwall merchant area
+    'private_key' => $GATEWAY['secretKey'] // available in your Paymentwall merchant area
+));
+
+$pingback = new Paymentwall_Pingback($_GET, $_SERVER['REMOTE_ADDR']);
+
+if ($pingback->validate()) {
+    $goodsId = isset($_GET['goodsid']) ? $_GET['goodsid'] : null;
+    $refId = isset($_GET['ref']) ? $_GET['ref'] : null;
+    $signature = isset($_GET['sig']) ? $_GET['sig'] : null;
```
Comment (sevastyanovio) [inline src/modules/gateways/callback/paymentwall.php]: $signature is not used

File: src/modules/gateways/paymentwall.php [hunk 4]
```diff
@@ -0,0 +1,92 @@
+<?php
+
+function paymentwall_config()
+{
+    $configarray = array(
+        "FriendlyName" => array("Type" => "System", "Value" => "Paymentwall"),
+        "appKey" => array("FriendlyName" => "Application Key", "Type" => "text", "Size" => "20",),
```
Comment (sevastyanovio) [inline src/modules/gateways/paymentwall.php]: Project key

**Pull request merged**